Makefile

```diff
@@ -17,31 +17,34 @@ TESTS := false
 # See "HTTPS Connection Support" in docs/endpoint.md for more information.
 ENFORCE_EP_HTTPS := false
 # The CFLAGS to pass during endpoint app build
```

Comment:
Clarify the description for LEGATO_FLAGS.

Reply:
At least, you should mention how it will be passed to Legato AF with mkapp involvement.

Reply:
Cc. @marktwtn 
Please be aware of the LEGATO_FLAGS  change.